README.md

```diff
@@ -1,4 +1,7 @@
-[![Netlify Status][Netlify-badge]][Netlify-url]
+[![Netlify Status][Netlify-badge]][Netlify-url] 
+<a href="https://discord.gg/dAaz472mPz">
```

Comment:
Use the markdown just like the netlify one!

[![discord-badge]][discord-url]

[discord-badge]: https://img.shields.io/discord/812156892343828500?logo=Discord
[discord-url]: https://discord.gg/dAaz472mPz


```
[![discord-badge]][discord-url]
```
Then at bottom:
```
[discord-badge]: https://img.shields.io/discord/812156892343828500?logo=Discord
[discord-url]: https://discord.gg/dAaz472mPz
```